src/backend/parser/Make.js

```diff
@@ -0,0 +1,26 @@
+var JisonLex = require('jison-lex');
+var Jison = require('jison');
+var fs = require('fs');
+var path = require('path');
+
+var dest = path.resolve(process.argv[2]); // first command-line argument
+
+try { // create directory if it does not exist
+    fs.mkdirSync(dest);
+} catch(e) {
+    if (e.code !== 'EEXIST' ) throw e;
+}
+
+console.log(process.argv[0] + ' ' + process.argv[1] + ' ' +process.argv[2]);
+var lexerFilename = path.join(dest, 'ansic.lex.js');
+var parserFilename = path.join(dest, 'ansic.js');
+
+var lexerGrammar = fs.readFileSync('src/backend/parser/ansic.jisonlex', 'utf-8');
```

Comment:
As for command line, I have explored the possibilities and I have come to the conclusion that doing it the 'node' way messes less with the user space. Compilers can be called from the command line, but then one has to choose between injecting them into user space(npm install -g) or calling them in Gruntfile.js via hardcoded path "node_modules/jison/...".